api/specs/webserver/openapi-projects.yaml

```diff
@@ -104,9 +104,9 @@ paths:
         default:
           $ref: "#/components/responses/DefaultErrorResponse"
 
-  /projects/{project_id}:
+  /projects/{project_uuid}:
```

Comment:
Are you sure about the name of in the spec? Would the webserver or something relying on these complaint? Like the frontend client?

Reply:
there is a lot of test coverage on that and only affects the tests
Take into account that this is is NOT a query parameter and since hte front-end is not really using the OAS, it will have no effect

Reply:
IN addition, I only changed it in the projects ... nowhere else

Reply:
On second thoughts ... perhaps it would be better, as you say to avoid so much noise. I will revert that change and do it in a separate PR